plugins/modules/flrtvc.py

```diff
@@ -360,7 +360,7 @@ def download(src, dst, resize_fs=True):
         False otherwise
     """
     res = True
-    wget = '/bin/wget'
+    wget = module.get_bin_path("wget", required=False, opt_dirs=['/opt/freeware/bin'])
```

Comment:
shouldn't the parameter `required` be True.
what happens if it can't find wget?